pyproject.toml

```diff
@@ -0,0 +1,31 @@
+name = "pyoorb"
+license = "GPL-3.0-only"
+
+[build-system]
+requires = ["setuptools", "wheel", "numpy"]
+
+[tool.cibuildwheel]
+before-build = """
+pip install numpy &&
+./configure gfortran opt --with-pyoorb --with-f2py=f2py --with-python=python
+"""
```

Comment:
Just to explain what's going on here: cibuildwheel lets you do pre-build work in two steps.

`before-all` is run just once for a target platform, across all Python versions. So this is used for installing system libraries.

`before-build` is run separately for each targeted Python version. So this is used when anything is Python version-dependent. That's true for us because f2py is linking to Python headers for a specific version.

So this config is used by every build. The ones below are platform-specific, eg using `yum` on the default linux builder, `brew` for the Mac one, and so on.